internal/server/handlers.go

```diff
@@ -502,6 +506,10 @@ func hasWildcard(s string) bool {
 	return strings.Contains(s, ">") || strings.Contains(s, "*")
 }
 
+func isValidName(name string) bool {
+	return !strings.ContainsAny(name, "\t\r\n\f.*>")
```

Comment:
space?

Reply:
Space is already checked before, sometimes it can be empty in case a user does not have an account